recipes/soups/potato-soup.adoc

```diff
@@ -0,0 +1,38 @@
+[id='sec.potato_soup']
+
+ifdef::env-github[]
+:imagesdir: ../../images
+endif::[]
+ifndef::env-github[]
+:imagesdir: images
+endif::[]
+
+
+=== Potato Soup
+
+.Ingredients
+[width="80%",cols="2,6,6",options="header"]
+|=========================================================
+|Metric | Ingredient | German/Zutat
+
+|400 ml | Water | Wasser
+|4 g / 0.4 pc | Stock cube | Brühwürfel
+|1 g / 0.5 leaf | Laurel | Lorbeer
+|5 ml | Paprika powder | Paprikapulver
+|300 g | Potatoes | Kartoffeln
+|50 g | Onion | Zwiebel
+|50 g | Wiener sausage | Wiener Würstchen oder Cabanossi
+|150 g | Red Pepper/Paprika | Rote Paprika
+|50 g | Heavy cream cheese | Doppelrahmkäse
```

Comment:
Cream cheese is "Frischkäse" which I assume you're not putting into the soup?

Reply:
I probably do:

https://lebensmittel-warenkunde.de/lebensmittel/milch-milchprodukte/kaese/frischkaese-und-speisequark/doppelrahmfrischkaese.html says

> dient der Doppelrahmfrischkäse als ideale Zutat für Soßen, **Suppen, Eintöpfe**, Aufläufe, Fondues und für Gebäck.